docs: add documentation on how to set up a online experiment via Fire… (#479)
AutoResearch/autora: docs/online-experiments/index.md

# Online Experiments To Collect AutoRA Observations

> AutoRA offers functionality to set up websites that integrate with an AutoRA workflow. These websites enable the utilization of observations gathered from participants online in a closed loop system.

Reviewer: ```suggestion
AutoRA provides functionality to set up websites that integrate with AutoRA workflows. These websites allow observations gathered from participants online to be used in a closed-loop system.
```
